compositCommon/AcquamanMPlot.pri

```diff
@@ -0,0 +1,103 @@
+
+PATH_TO_ACQUAMAN = /home/liux0/beamline/programming/acquaman/source
```

Comment:
Is this essentially the core pieces from acquamanCommon.pri but as an includable pri file?